libopenage/console/console.cpp

```diff
@@ -113,6 +119,10 @@ void Console::interpret(const std::string &command) {
 	if (command == "exit") {
 		this->set_visible(false);
 	}
+	else if (command == "log-tail") {
+		for (auto it = tailsink->tail.begin(); it != tailsink->tail.end(); it++)
+			this->buf.write((*it).c_str());
```

Comment:
braces here as well pls, and you can use `auto &it : this->tailsink`. and `it->c_str()`.